File: PluginDemo/app/src/main/java/com/wyl/plugindemo/bean/ControlAttribute.java
package com.wyl.plugindemo.bean;

/**
 * Created by wyl on 2020/1/17.
 */
public class ControlAttribute {
    private String text;//文本内容
    private String textColor;//#2232323
    private float textSize = 15;//文字大小
    private String backgroundImage;//背景图片 连接形式
    private String backgroundColor;//背景颜色  #12312312
    private int gravity;//文字方向 1 居中

    public float getTextSize() {
        return textSize;
    }

    public void setTextSize(float textSize) {
        this.textSize = textSize;
    }

    public int getGravity() {
        return gravity;
    }

    public void setGravity(int gravity) {
        this.gravity = gravity;
    }

    public String getText() {
        return text == null ? "" : text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getTextColor() {
        return textColor == null ? "#000000" : textColor;
    }

    public void setTextColor(String textColor) {
        this.textColor = textColor;
    }


    public String getBackgroundImage() {
        return backgroundImage == null ? "" : backgroundImage;
    }

    public void setBackgroundImage(String backgroundImage) {
        this.backgroundImage = backgroundImage;
    }

    public String getBackgroundColor() {
        return backgroundColor == null ? "#ffffff" : backgroundColor;
    }

    public void setBackgroundColor(String backgroundColor) {
        this.backgroundColor = backgroundColor;
    }
}

File: InformationReleaseSystem/ReleaseSystem/bean/ControlAttribute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReleaseSystem.bean
{
    public class ControlAttribute
    {
        public String text { get; set; }//文本内容
        public String textColor { get; set; }//#2232323
        public float textSize { get; set; }//文字大小
        public String backgroundImage { get; set; }//背景图片 连接形式
        public String backgroundColor { get; set; }//背景颜色  #12312312
        public int gravity { get; set; }//文字方向 1 居中


    }
}
